fighters/kirby/src/status/mod.rs

```diff
@@ -31,6 +39,12 @@ unsafe extern "C" fn change_status_callback(fighter: &mut L2CFighterCommon) -> L
         VarModule::set_int(fighter.battle_object, vars::koopa::instance::SPECIAL_N_FIREBALL_COOLDOWN, KOOPA_MAX_COOLDOWN);
     }
 
+    /// Re-enables Stone
+    if fighter.is_situation(*SITUATION_KIND_GROUND) || fighter.is_situation(*SITUATION_KIND_CLIFF) 
+    || fighter.is_status_one_of(&[*FIGHTER_STATUS_KIND_REBIRTH, *FIGHTER_STATUS_KIND_DEAD, *FIGHTER_STATUS_KIND_LANDING, *FIGHTER_STATUS_KIND_GIMMICK_SPRING_JUMP]) {
```

Comment:
Remove the checks for `REBIRTH` and `DEAD` as they already have this flag reset in their statuses below. You also don't need `LANDING` as the `SITUATION_KIND_GROUND` check should cover that

Reply:
We actually do need to check for LANDING as well. Both are included in our other resource refresh checks. There is some weirdness with landing + edge canceling within 1f that won’t always update your situation kind.

Reply:
I hate it here